### PR TITLE
docs: add build-in pipeline greptime_identity doc

### PR DESCRIPTION
**Pull request by paomian**
## What's Changed in this PR

* add build-in pipeline greptime_identity doc

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/user-guide/logs/quick-start.md [hunk 1]
```diff
@@ -138,6 +138,28 @@ You will see the following output if the command is successful:
 {"output":[{"affectedrows":4}],"execution_time_ms":79}
 ```
 
+### Write logs useing GreptimeDB built-in Pipeline
```
Comment (nicecui) [inline docs/user-guide/logs/quick-start.md:141]:
```suggestion
### Write logs using GreptimeDB built-in Pipeline
```

File: docs/user-guide/logs/write-logs.md [hunk 2]
```diff
@@ -102,6 +102,24 @@ processors:
 
 It is recommended to use `dissect` or `regex` processor to split the input line into fields first and then process the fields accordingly.
 
+## Build-in Pipeline
+
+Currently we only have a built-in pipeline called `greptime_identity` that stores user input as is. We'll add an additional `greptime_timestamp` to the user data to indicate when it was written. We'll convert the json data into our table structure, and our conversion follows a few rules
```
Comment (nicecui) [inline docs/user-guide/logs/write-logs.md:107]: do not need to write the `greptime_timestamp` now

File: docs/user-guide/logs/write-logs.md [hunk 3]
```diff
@@ -102,6 +102,24 @@ processors:
...
+
+1. we will scan all the records in a batch of data and get a superset of the list of fields to be considered as the table schema of the batch. please be careful to keep the data fields fixed as much as possible. 2. when the same field types are not the same, we will scan the data for all the records.
```
Comment (nicecui) [inline docs/user-guide/logs/write-logs.md:109]: what does the superset mean? can you explain more?

**Comment by shuiyisong** [comment]
In my opinion, `greptime_identity` is more like 'try to recognize the JSON-formatted data's schema and transform them accordingly' rather than a built-in pipeline. The pipeline name is just a way to enter this mode, it's not actually a pipeline.

We can say the `greptime_` prefix of in pipeline name is reserved.

File: docs/user-guide/logs/write-logs.md [hunk 4]
```diff
@@ -102,6 +102,67 @@ processors:
...
 
+## Build-in Pipeline
+
+Currently we only have a built-in pipeline called `greptime_identity` that stores user input as is. We'll add an additional `greptime_timestamp` to the user data to indicate when it was written. We'll convert the json data into our table structure, and our conversion follows a few rules
+
+1. We will scan all the records in a batch of data and get a superset of the list of fields to be considered as the table schema of the batch. please be careful to keep the data fields fixed as much as possible.
+2. An error is returned when the same field has a different type on a different line.
+3. We ignore all fields with null value.
+
+### Type conversion rules
```
Comment (nicecui) [inline docs/user-guide/logs/write-logs.md:113]:
```suggestion
## Built-in Pipelines

GreptimeDB offers built-in pipelines for common log formats, allowing you to use them directly without creating new pipelines.

### `greptime_identity`

The `greptime_identity` pipeline is designed for writing JSON logs and automatically creates columns for each field in the JSON log.

- The first-level keys in the JSON log are used as column names.
- An error is returned if the same field has different types.
- Fields with `null` values are ignored.
- An additional column, `greptime_timestamp`, is added to the table as the time index to indicate when the log was written.

#### Type conversion rules
```

Comment (nicecui) [reply docs/user-guide/logs/write-logs.md:113]: please move this section to `manage-pipelines.md`, the location would be before `## Create a Pipeline`

**Comment by paomian** [comment]
> In my opinion, `greptime_identity` is more like 'try to recognize the JSON-formatted data's schema and transform them accordingly' rather than a built-in pipeline. The pipeline name is just a way to enter this mode, it's not actually a pipeline.
> 
> We can say the `greptime_` prefix of in pipeline name is reserved.

It's true that this is not a pipeline from an implementation point of view, but thinking of it as a pipeline helps reduce the cost of understanding. It's something else that doesn't matter what it's called, but it can be used to transform user-submitted json data as is, just like a pipeline.

File: docs/user-guide/logs/quick-start.md [hunk 5]
```diff
@@ -52,6 +52,30 @@ Refer to [Write Data Using gRPC](/user-guide/ingest-data/for-iot/grpc-sdks/overv
...
+- `table=pipeline_logs` specifies the target table. If the table does not exist, it will be created automatically.
+The `greptime_identity` pipeline will automatically create columns for each field in the JSON log. A successful command execution will return:
+
+```json
+{"output":[{"affectedrows":3}],"execution_time_ms":9}
+```
+
+For more details about the `greptime_identity` pipeline, please refer to the [Manage Pipelines](manage-pipelines.md#greptime_identity) document.
+
 ### Create a Pipeline
```
Comment (nicecui) [inline docs/user-guide/logs/quick-start.md:79]:
```suggestion

### Write logs using a custom Pipeline

If your logs follow a specific pattern, you can create a custom pipeline to parse and transform the log messages into multiple columns, and automatically create tables.

#### Create a Pipeline
```

Comment (nicecui) [reply docs/user-guide/logs/quick-start.md:79]: please also rewrite `### Write logs` to `#### Write logs` in the following content

File: docs/user-guide/logs/write-logs.md [hunk 6]
```diff
@@ -100,8 +100,4 @@ processors:
 # rest of the file is ignored
 ```
 
-It is recommended to use `dissect` or `regex` processor to split the input line into fields first and then process the fields accordingly.
-
-## Example
-
-Please refer to the "Writing Logs" section in the [Quick Start](quick-start.md#write-logs) guide for an example.
```
Comment (nicecui) [inline docs/user-guide/logs/write-logs.md:107]: please do not delete this section

**Review by nicecui — COMMENTED** [review]
LGTM, please add the zh content

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md [hunk 7]
```diff
@@ -5,6 +5,73 @@
 
 有关 Pipeline 的具体配置，请阅读 [Pipeline 配置](pipeline-config.md)。
 
+## 内置 Pipeline
+
+GreptimeDB 提供了常见日志格式的内置 Pipeline，允许您直接使用而无需创建新的 Pipeline。
+
+请注意，内置 Pipeline 不可编辑。此外，Pipeline 名称的 "greptime_" 前缀是保留的。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md:12]:
```suggestion
请注意，内置 Pipeline 的名称以 "greptime_" 为前缀，不可编辑。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md [hunk 8]
```diff
@@ -5,6 +5,73 @@
...
+
+### `greptime_identity`
+
+`greptime_identity` Pipeline 适用于写入 JSON 日志，并自动为 JSON 日志中的每个字段创建列。
+
+- JSON 日志中的一级键用作列名。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md:18]:
```suggestion
- JSON 日志中的第一层级的 key 是表中的列名。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md [hunk 9]
```diff
@@ -5,6 +5,73 @@
...
+- 如果相同字段具有不同类型，则会返回错误。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md:19]:
```suggestion
- 如果相同字段包含不同类型的数据，则会返回错误。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md [hunk 10]
```diff
@@ -5,6 +5,73 @@
...
+
+```json
+[
+    {"name": "Alice", "age": 20, "is_student": true, "score": 90.5,"object": {"a":1,"b":2}},
+    {"age": 21, "is_student": false, "score": 85.5, "company": "A" ,"whatever": null},
+    {"name": "Charlie", "age": 22, "is_student": true, "score": 95.5,"array":[1,2,3]}
+]
+```
+
+我们将合并每个批次的行模式以获得最终模式。表模式将如下所示：
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/manage-pipelines.md:42]:
```suggestion
我们将合并每个批次的行结构以获得最终 schema。表 schema 如下所示：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/quick-start.md [hunk 11]
```diff
@@ -50,7 +50,33 @@ INSERT INTO origin_logs (message, time) VALUES
...
+- `table=pipeline_logs` 指定了目标表。如果表不存在，将自动创建。
+`greptime_identity` pipeline 将自动为 JSON 日志中的每个字段创建列。成功执行命令将返回：
+
+```json
+{"output":[{"affectedrows":3}],"execution_time_ms":9}
+```
+
+有关 `greptime_identity` pipeline 的更多详细信息，请参阅 [管理 Pipeline](manage-pipelines.md#greptime_identity) 文档。
+
+### 使用 Pipeline 写入日志
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/logs/quick-start.md:77]:
```suggestion
### 使用自定义 Pipeline 写入日志
```

**Review by nicecui — APPROVED** [review]

**Review by fengjiachun — APPROVED** [review]
LGTM

**Pull request merged**